Asterisk.2013/Asterisk.NET/Manager/Event/AorDetailEvent.cs

```diff
@@ -0,0 +1,10 @@
+namespace AsterNET.Manager.Event
+{
+	public class AorDetailEvent : ResponseEvent
+	{
+		public AorDetailEvent(ManagerConnection source)
+			: base(source)
+		{
+		}
```

Comment:
Could add some properties to this class.
Asterisk documentation has a list of the values.
See: AorDetail